base_layer/core/src/base_node/chain_metadata_service/service.rs

```diff
@@ -85,19 +88,27 @@ impl ChainMetadataService {
             tokio::select! {
                 Ok(block_event) = block_event_stream.recv() => {
                     log_if_error!(
-                        level: debug,
+                        level: warn,
                         target: LOG_TARGET,
                         "Failed to handle block event because '{}'",
                         self.handle_block_event(&block_event).await
                     );
                 },
 
                 Ok(event) = liveness_event_stream.recv() => {
-                    log_if_error!(
-                        target: LOG_TARGET,
-                        "Failed to handle liveness event because '{}'",
-                        self.handle_liveness_event(&event).await
-                    );
+                    match
+                        self.handle_liveness_event(&event).await {
+                        Ok(_) => {}
+                        Err(e) => {
+                           error!( target: LOG_TARGET, "Failed to handle liveness event because '{}'", e);
+                           if let ChainMetadataSyncError::ReceivedInvalidChainMetadata(node_id,reason) = e {
+                               log_if_error!(
+                                 level: warn,
```

Comment:
Same here. Banning a peer is informational not deserving of anything alerting to an end user.